engine/installation/linux/ubuntulinux.md

```diff
@@ -116,365 +117,430 @@ packages from the Docker repository:
+1.  Open a terminal on your Ubuntu host.
 
-2. Update your package manager.
+2.  Update your package manager.
 
-        $ sudo apt-get update
+    ```bash
+    $ sudo apt-get update
+    ```
 
-3. Install both the required and optional packages.
+3.  Install both the required and optional packages.
 
-        $ sudo apt-get install linux-image-generic-lts-trusty
+    ```bash
+    $ sudo apt-get install linux-image-generic-lts-trusty
+    ```
 
-    Depending on your environment, you may install more as described in the preceding table.
+    Repeat this step for other packages you need to install.
 
-4. Reboot your host.
+4.  Reboot your host to use the updated kernel.
 
-        $ sudo reboot
+    ```bash
+    $ sudo reboot
+    ```
 
-5. After your system reboots, go ahead and install Docker.
+5.  After your system reboots, go ahead and
+    [install Docker](ubuntulinux.md#install).
 
 ## Install
 
-Make sure you have installed the prerequisites for your Ubuntu version.
-
-Then,
-install Docker using the following:
+Make sure you have satisfied all the
+[prerequisites](ubuntulinux.md#prerequisites), then follow these steps.
 
-1. Log into your Ubuntu installation as a user with `sudo` privileges.
+1.  Log into your Ubuntu installation as a user with `sudo` privileges.
 
-2. Update your `APT` package index.
+2.  Update your `APT` package index.
 
-        $ sudo apt-get update
-
-3. Install Docker.
+    ```bash
+    $ sudo apt-get update
+    ```
+3.  Install Docker.
 
-        $ sudo apt-get install docker-engine
+    ```bash
+    $ sudo apt-get install docker-engine
+    ```
 
-4. Start the `docker` daemon.
+4.  Start the `docker` daemon.
 
-        $ sudo service docker start
+    ```bash
+    $ sudo service docker start
+    ```
 
-5. Verify `docker` is installed correctly.
+5.  Verify that `docker` is installed correctly by running the `hello-world`
+    image.
 
-        $ sudo docker run hello-world
+    ```bash
+    $ sudo docker run hello-world
+    ```
 
     This command downloads a test image and runs it in a container. When the
-    container runs, it prints an informational message. Then, it exits.
+    container runs, it prints an informational message and exits.
 
 ## Optional configurations
 
-This section contains optional procedures for configuring your Ubuntu to work
-better with Docker.
+This section contains optional procedures for configuring Ubuntu to work better
+with Docker.
 
-* [Create a docker group](ubuntulinux.md#create-a-docker-group)
+* [Create a `docker` group](ubuntulinux.md#create-a-docker-group)
 * [Adjust memory and swap accounting](ubuntulinux.md#adjust-memory-and-swap-accounting)
 * [Enable UFW forwarding](ubuntulinux.md#enable-ufw-forwarding)
 * [Configure a DNS server for use by Docker](ubuntulinux.md#configure-a-dns-server-for-use-by-docker)
 * [Configure Docker to start on boot](ubuntulinux.md#configure-docker-to-start-on-boot)
 
-### Create a Docker group
+### Create a `docker` group
```

Comment:
See my earlier comment on back-ticks in headings https://github.com/docker/docker.github.io/pull/640/files#r88769224 (wasn't sure if you missed it, or decided to keep it as-is :smile:)

Perhaps better; describe what we're trying to achieve (e.g. "run docker commands without sudo", "run docker commands as a non-privileged user". Need to be careful picking the right words to not have users assume the docker _daemon_ can be run as a non-privileged user)